spinta/types/type.py

```diff
@@ -33,7 +34,12 @@ def is_valid(self, value: typing.Any):
 class PrimaryKey(Type):
 
     def load(self, value: typing.Any):
-        return str(value)
+        try:
+            # is it possible to get ValueError when doing string conversion?
+            return str(value)
+        except ValueError:
+            raise DataError(f'Invalid value for pk type: {value}')
```

Comment:
I don't think it is possible to get ValueError from `str(value)`, so I would add error handlers only for code proven with tests.